.gitignore

```diff
@@ -12,9 +12,9 @@
 /.idea/gradle.xml
 /.idea/misc.xml
 /.idea/jarRepositories.xml
+/.idea/codeStyles
 .idea/workspace.xml
 .idea/tasks.xml
-.idea/gradle.xml
```

Comment:
Why idea file changes?  If there are changes to idea files please give pr to master.